source/constructs/utils/utils.ts

```diff
@@ -41,3 +43,24 @@ export function addCfnCondition(resource: Resource | CfnResource | undefined, co
 
   resource.cfnOptions.condition = condition;
 }
+
+/**
+ * Run a shell command from typescript
+ * @param shellCommand Command to run in the shell
+ * @param pathToRunCommandIn CWD of shell command
+ */
+export function RunShellCommand(shellCommand: string, pathToRunCommandIn: string) {
```

Comment:
For consistency sake, let's use `lowerCamelCase` for function names

Reply:
As always, excellent comments and eye for details. We will take care of them. tks